packages/appeal-reply-service-api/README.md

```diff
@@ -0,0 +1,53 @@
+# Appeals Service API
```

Comment:
Update with correct details - this is a copy/paste of the Appeals Service API

Reply:
Done.